config/locales/en.yml

```diff
@@ -41,6 +41,7 @@ en:
               taken: "This certificate has already been uploaded"
               blank: "No Certificate file selected. Please choose a file to try again."
               invalid_certificate: "Certificates must be in a .PEM format"
+              no_parent: "The certificate does not have a known issuing certificate. Please add its issuing certificate first."
```

Comment:
To be consistant, you may want to move some of the other messages, like those in certificate_controller into there?

Reply:
👍